components/movementsensor/wheeledodometry/wheeledodometry.go

```diff
@@ -248,9 +259,22 @@ func (o *odometry) Orientation(ctx context.Context, extra map[string]interface{}
 func (o *odometry) CompassHeading(ctx context.Context, extra map[string]interface{}) (float64, error) {
 	o.mu.Lock()
 	defer o.mu.Unlock()
+
+	if o.useOri {
+		return yawToCompassHeading(o.orientation.Yaw), nil
```

Comment:
yaw returns in radians from -pi -> pi, so wrote a function to convert this to 0-360